### DISCUSSION TITLE
Add allowed chat id using Filters.Chat.add_chat_ids()

### DISCUSSION THREAD

**Discussion by ajuancer** [python-telegram-bot/python-telegram-bot#4092] [Q&A]
Hi!

I have the following [filter](https://docs.python-telegram-bot.org/en/v20.7/telegram.ext.filters.html#telegram.ext.filters.Chat.add_chat_ids),
```py
test = filters.Chat
test.add_chat_ids(chat_id)
```
wich gives:
```c
TypeError: add_chat_ids() missing 1 required positional argument: 'chat_id'
```

**Comment by Bibo-Joshi** [comment]
If you carefully read the docs of [`filters.Chat`](https://docs.python-telegram-bot.org/en/v20.7/telegram.ext.filters.html#telegram.ext.filters.Chat) you will find that they include an example of how you can build an instance of that filter.

**Comment by ajuancer** [comment]
Hi again,
So the problem I was facing with `Chat()` was that the bot was not responding at all... the problem was the number value was stored as string because I was importing it from another file. Just casting the value to int solves the problem. Here a short snippet for future reference:

```py
test = filters.Chat(int(chat_id))
```
Anyways I would like to know what was the problem with the use of `add_chat_ids()`

Happy coding!
